src/components/filters/polygon-filter.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2022 Uber Technologies, Inc.
```

Comment:
this file is treated as new file, instead of a rename

Reply:
fixed